extensions/BMO/lib/Reports/WhatsNext.pm

```diff
@@ -146,16 +148,19 @@ sub s1_bugs {
   my $user = shift;
   my $dbh  = Bugzilla->dbh;
 
-  my $query = SELECT . '
+  # Preselected values for inserting into SQL
+  my $cache      = Bugzilla->request_cache->{whats_next};
```

Comment:
A request-cache isn't sufficient; as per my previous review it needs to be **per-process or similar**.